src/openrct2/ride/coaster/meta/GigaCoaster.h

```diff
@@ -54,7 +54,7 @@ constexpr RideTypeDescriptor GigaCoasterRTD =
     )),
     SET_FIELD(ColourPreview, { SPR_RIDE_DESIGN_PREVIEW_GIGA_COASTER_TRACK, SPR_RIDE_DESIGN_PREVIEW_GIGA_COASTER_SUPPORTS }),
     SET_FIELD(ColourKey, RideColourKey::Ride),
-    SET_FIELD(Name, "giga_rc"),
+    SET_FIELD(Name, "GIGA_RC"),
```

Comment:
Looks like an accidental revert.

```suggestion
    SET_FIELD(Name, "giga_rc"),
```

Reply:
yes i made a mistake back then i had no stats left on my coaster and had deleted the gigabahn for whatever reason. i think i wanted to replace something i clicked replace all but not only in the current form but in the whole project.